website/docs/r/bigip_ltm_snat.html.markdown

```diff
@@ -23,6 +23,7 @@ resource "bigip_ltm_snat" "snat3" {
   origins = ["6.1.6.6"]
   mirror = "false"
   snatpool = "/Common/sanjaysnatpool"
+  vlans = ["test-vlan"]
```

Comment:
I'm confused, if `vlans` is a `TypeSet` schema:
https://github.com/terraform-providers/terraform-provider-bigip/blob/001837390aba76e5deee2c7ca0c0747d823bcafd/bigip/resource_bigip_ltm_snat.go#L70-L76

Why are you using a List definition here?